internal/domain/permission.go

```diff
@@ -28,7 +28,7 @@ type PermissionCheck func(ctx context.Context, permission, orgID, resourceID str
 
 const (
 	PermissionUserWrite     = "user.write"
-	PermissionSessionRead   = "session.read"
```

Comment:
Why is `SessionRead` replaced?

Reply:
i think my IDE complained that it is not used